app/2.0/docs/tools/create-app-polymer-cli.md

```diff
@@ -0,0 +1,102 @@
+CLI comes with a `basic` template, which is the most basic starting point for a Polymer-based 
+application, as well as others that introduce more complex layout and application patterns.
+
+This chapter teaches you more about `basic` app projects.  See [Polymer App Toolbox 
+templates](../../toolbox/templates) for more details on other templates.
+
+## App project architecture {#app-architecture}
+
+Polymer CLI is designed for apps that follow the [app shell 
+architecture](https://developers.google.com/web/updates/2015/11/app-shell).
+
+There are fundamental concepts of the app shell architecture that you should understand before 
+creating your app project with Polymer CLI: the entrypoint, the shell, and fragments. See [App 
+structure](../../toolbox/prpl#app-structure) from the App Toolbox docs for an in-depth overview of 
+these concepts.
+
+## Set up basic app project {#basic-app}
+
+Follow the steps below to get your `basic` app project set up.
+
+1.  Create a directory for your app project.
+
+    <pre><code>mkdir <var>app</var>
+    cd <var>app</var></code></pre>
+
+    Where <code><var>app</var></code> is the name of your project directory.
+
+1.  Initialize your app. Polymer CLI asks you a few questions
+    as it sets up your app.
+
+        polymer init
+
+1.  Select `polymer-2-application`.
+
+1.  Enter a name for your app. Defaults to the name of the current directory.
+
+1.  Enter a name for the main element in your project. The main element is the
+    top-most, application-level element of your app. Defaults to the name of
+    the current directory, followed by `-app`.
+
+    The code samples throughout this doc use the example app element name
+    <code><var>my-app</var></code>. When creating your app you'll want to
+    replace any instance of <code><var>my-app</var></code> with the name of
+    your main element.
+    {.alert .alert-info}
+
+1.  Enter a description for your app.
+
+At this point, Polymer CLI generates files and directories for your app, and installs your 
+project's dependencies.
+
+## App project layout
+
+After the initialization process Polymer CLI generates the following files and directories.
+
+*   `bower.json`. Configuration file for Bower.
+*   `bower_components/`. Project dependencies. See [Manage dependencies](#dependencies).
+*   `index.html`. Entrypoint page of the app.
+*   `src/`<code><var>my-app</var></code>/<code><var>my-app</var></code>`.html`.
+    Source code for main element.
+*    `test/`<code><var>my-app</var></code>/<code><var>my-app</var></code>`_test.html`. Tests for 
+main element.
+
+### Add elements
+
+You may want to compose your main element out of smaller, application-specific elements. These 
+app-specific elements should be defined in the `src` directory, at the same level as 
+<code><var>my-app</var></code>.
+
+    app/
+      src/
+        my-app/
+        my-el/
+
+Currently there is no Polymer CLI command to generate application-specific elements. You should do 
```

Comment:
I think we kind of need to highlight this to prevent people from trying to do init element inside an app project. But instead of saying, "There is currently no..." We could say,

To add another element to the project:

1. Create a new folder under src.

        mkdir src/my-el

2. Create an HTML import for the new element. You can use the existing app element as a starting point.

3. To use the new element, you'll need to import it into your application element.

 Don't use `polymer init` to create an element project inside your app project.

This adds more detail without being entirely satisfying (i.e., doesn't show adding the `<link rel="import">` or using the element in the app element). So alternately, just add the .html files to the diagram above for clarity:


```
my-example/
     src/
       my-example-app/
         my-example-app.html
       my-el/
         my-el.html
```

You can copy the existing app element as a starting point for your next element.

 Don't use `polymer init` to create an element project inside your app project.

=== end suggestion

I changed the example here to "my-example" so the directory names would make sense. If you're using `my-app` as a placeholder, should be `<var>my-app</var>` by style. Downside: examples might need to be formatted in HTML instead of fenced code blocks, because fenced code blocks escape the HTML markup.